### ISSUE TITLE
0.12 release candidates

### ISSUE THREAD

**Issue by necolas** [necolas/react-native-web#1512]
I published `0.12.0-rc.1` which has all the changes that are likely to go into 0.12.0 unless issues are spotted. The possible exception being the issue left in the [0.12 milestone](https://github.com/necolas/react-native-web/milestone/10) but I'll need to check a possible change in IE11 and then have people check their app layouts didn't break.

cc @EvanBacon @paularmstrong @comp615 @brunolemos

**Comment by brunolemos** [comment]
I noticed only one small layout issue. The `placeholderTextColor` is not working:

![gif](https://user-images.githubusercontent.com/619186/71298238-ee9a9a80-2365-11ea-8623-03a5530c5e30.gif)

And because of the `ToolbarAndroid` removal I had to make this PR: https://github.com/software-mansion/react-native-gesture-handler/pull/888

**Comment by necolas** [comment]
@EvanBacon I won't be maintaining RN deprecated APIs in RNW anymore, so if Expo users need to rely on deprecated exports like `ViewPropTypes` you will either need to provide them yourselves or use `babel-plugin-transform-react-remove-prop-types` in the webpack config to strip PropTypes from dependencies.

**Comment by vladp** [comment]
We just a test upgrade from RNW 0.11.7 to 0.12.0-rc1.


The code base is close to 30K lines (so not small and not big). For RN we use 0.60

We are reasonably conservative users of RN/RNW so no special/unusual usage.
But was happy to see that things like flatlist, buttons and other 'easy-but-useful' RN apis remained in RNW 0.12.

We also use several RN-but-works-on-RNW packages like react-native-fit-image, react-native-vector-icons, expo, expo-av, and a few others.

Out of the above, the only problem was expo/expo-av due to ViewPropTypes mentioned above.
Temporarily, we removed expo-av from the code base and dropped back to browser's `<  audio >` tag.

There were no other problems at all. 

So  thank you @necolas  for making this upgrade easy.

Hoping that expo team can address the ViewPropTypes issue.

We also saw reduction of gzipped code size after the migration (10kb saving.. which is pretty good), compared to RN 11.7

```
rnw -0.12.0-rc1
  276.11 KB  build\static\js\2.dbdd813d.chunk.js
  77.88 KB   build\static\js\main.b75b01a6.chunk.js
  23.73 KB   build\static\css\2.42c6d710.chunk.css
  736 B      build\static\js\runtime-main.049aa418.js
  705 B      build\static\css\main.9ad727fb.chunk.css

```
```

rnw -0.11.7

  285.96 KB (+9.85 KB)  build\static\js\2.a91114c4.chunk.js
  77.8 KB (-81 B)       build\static\js\main.d0bd6c3e.chunk.js
  23.73 KB              build\static\css\2.42c6d710.chunk.css
  736 B                 build\static\js\runtime-main.049aa418.js
  705 B                 build\static\css\main.9ad727fb.chunk.css

```

**Comment by necolas** [comment]
Closed by 0.12.0 https://github.com/necolas/react-native-web/commit/dee418bb930fdeabfad8361a2fdc4247519638a9

**Comment by zamponotiropita** [comment]
`babel-plugin-transform-react-remove-prop-types` breaks `StackView.tsx` from `react-navigation-stack`
